docs/Documentation/deployment_guide_introduction.md

```diff
@@ -4,8 +4,8 @@
 Once you have a model that meets both OpenVINO™ and your requirements, you can choose among several ways of deploying it with your application: 
 
 * [Deploy your application locally](../OV_Runtime_UG/deployment/deployment_intro.md).
-* [Deploy your model online with the OpenVINO Model Server](@ref ovms_what_is_openvino_model_server).
-* [Deploy your application under the TensorFlow framework with, OpenVINO Integration](./openvino_ecosystem_ovtf.md).
+* [Deploy your model with OpenVINO Model Server](@ref ovms_what_is_openvino_model_server).
+* [Deploy your application under the TensorFlow framework with OpenVINO Integration](./openvino_ecosystem_ovtf.md).
```

Comment:
This is not changed but now that this is being edited: "deploy under" reads strange to me. I would expect with/for/to/in.